### ISSUE TITLE
ImportError: numpy.core.multiarray failed to import

### ISSUE THREAD

**Issue by kishoreKunisetty** [online-ml/river#831]
## Versions

**river version**: 0.9.0
**Python version**: 3.7.12
**Operating system**: linux

## Describe the bug

I tried to replicate the example given in river website [SNARIMAX ](https://riverml.xyz/latest/api/time-series/SNARIMAX/) in colab
then during import itself facing this issue

## Steps/code to reproduce

```python
# Sample code to reproduce the problem
!pip install river

from river import compose
from river import datasets
from river import linear_model
from river import metrics
from river import optim
from river import preprocessing
from river import time_series
# Please do your best to provide a Minimal, Reproducible Example: https://stackoverflow.com/help/minimal-reproducible-example
```

**Comment by MaxHalford** [comment]
Hey! I suggest reading #815 :)

**Comment by MaxHalford** [comment]
This should be fixed with River 0.10.0 :)
